src/System.Collections.Immutable/src/System.Collections.Immutable.csproj

```diff
@@ -9,16 +9,20 @@
     <FileAlignment>512</FileAlignment>
     <DocumentationFile>$(OutputPath)$(MSBuildProjectName).xml</DocumentationFile>
     <GenerateAppxPackageOnBuild>False</GenerateAppxPackageOnBuild>
-    <PackageTargetFramework Condition="'$(TargetGroup)' == 'netstandard1.0'">netstandard1.0;portable-net45+win8+wp8+wpa81</PackageTargetFramework>
```

Comment:
What was this removed? I think this might remove the old-style portable version of this library in the nuget package

Reply:
Chatted with @safern. will send another PR now to add it back.